packages/react/src/slider/value/SliderValue.tsx

```diff
@@ -29,9 +29,10 @@ export const SliderValue = React.forwardRef(function SliderValue(
 
   const outputFor = React.useMemo(() => {
     let htmlFor = '';
-    for (const thumbMetadata of thumbMap.values()) {
-      if (thumbMetadata?.inputId) {
-        htmlFor += `${thumbMetadata.inputId} `;
+    for (const thumbElement of thumbMap.keys()) {
+      if (thumbElement instanceof Element) {
```

Comment:
`isElement` is a local util

If there are other `instanceof`s, they should be updated as well since I don't think this check works in iframes/different doc contexts